tests/models/flow_cell/test_flowcell_model.py

```diff
@@ -175,3 +176,29 @@ def test_has_demultiplexing_started_on_sequencer_false(
 
     # THEN the response should be False
     assert not has_demux_started
+
+
+def test_sequencing_dates_novaseqx_flow_cell(novaseq_x_flow_cell_dir: Path):
+    # GIVEN a flow cell directory data for a novaseq x flow cell
+    flow_cell = FlowCellDirectoryData(novaseq_x_flow_cell_dir)
```

Comment:
you can use the flow cell fixture `novaseq_x_flow_cell` directly so you don't need to initialise one. (`tests/fixture_plugins/demultiplex_fixtures/flow_cell_fixtures.py`)